app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, dist: Distance | int) -> Distance:
+        if isinstance(dist, Distance):
+            return Distance(self.km + dist.km)
+        return Distance(self.km + dist)
+
+    def __iadd__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, dist: int) -> Distance:
+        return Distance(self.km * dist)
+
+    def __truediv__(self, dist: int) -> Distance:
+        return Distance(round(self.km / dist, 2))
```

Comment:
Can be error, but there are not cases in task how to be in that case. I don't know what to do in this case. Return self or division by 1  instead of 0? Raise exception ? We haven't done this theme